R/drive_upload.R

```diff
@@ -21,58 +21,77 @@
 #' @export
 #' @examples
 #' \dontrun{
-#' write.csv(chickwts, "chickwts.csv")
-#' drive_chickwts <- drive_upload("chickwts.csv")
+#' ## upload a csv file
+#' mirrors_csv <- drive_upload(R.home('doc/BioC_mirrors.csv'))
 #'
 #' ## or convert it to a Google Sheet
-#' drive_chickwts <- drive_upload("chickwts.csv", type = "spreadsheet")
+#' mirrors_sheet <- drive_upload(
+#'   R.home('doc/BioC_mirrors.csv'),
+#'   name = "BioC_mirrors",
+#'   type = "spreadsheet"
+#' )
+#'
+#' ## check out the new Sheet!
+#' drive_browse(mirrors_sheet)
+#'
+#' ## clean-up
+#' drive_search("BioC_mirrors") %>% drive_delete()
 #' }
-drive_upload <- function(from = NULL,
-                         name = NULL,
+drive_upload <- function(file = NULL,
```

Comment:
I see no reason this can't be `file` and match lots of other function signatures. Right?